content/work/timedesigner.md

```diff
@@ -81,15 +114,403 @@ press:
     <<: *furman
     slug: process
 summary: |
-  Tegy TimeDesigner helps school teams
-  easily create multiple scheduling scenarios
+  TimeDesigner helps school teams
+  rapidly create multiple scheduling scenarios
   from various perspectives,
   and plan ahead for future iterations.
-  Currently the software is only available to schools in
+  OddBird helped Tegy
+  plan, design, develop, launch, and maintain
+  their school scheduling web app
+  using CSS grid and custom property wizardry
+  on the front-end,
+  and well-tested Django/Python on the back-end.
+  Currently, the software is only available to schools in
   Tegy training programs.
 ---
 
 {% import 'quotes.macros.njk' as quotes %}
+{% import 'embed.macros.njk' as embed %}
+{% import 'layout.macros.njk' as layout %}
 
-<!-- all but the first quote -->
-{{ quotes.grid(press | slice(2) | first ) }}
+{{ quotes.find(collections.all, ['range', 'scheduling']) }}
+
+{{ layout.title('What We Did') }}
+
+{% call embed.media_block(
+  media=embed.screen(
+    notch='0',
+    content=embed.img(
+      src='work/timedesigner/mobile-lg.jpg',
+      alt='small screen showing schedule grid filled with items',
+      sizes='media'
```

Comment:
see above comment…

Reply:
This one will be easy to update when the Work Summaries branch is merged. I'm not sure about the previous comment or if that macro will need to be updated first.

Reply:
@stacyk I merged `work-summaries` into this PR -- feel free to do final cleanup.